### PR TITLE
Etch-a-Sketch Project

### PR DESCRIPTION
**Pull request by Sushilkumar3255**
etch-1
etcch-2
etch-3

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Etch-a-Sketch Dashboard</title>
+    <link rel="stylesheet" href="./style.css">
+    <!-- <script src="./sketch.js" defer></script> -->
```
Comment (roopaljasnani) [inline index.html]: pls make sure not  to have any commented code

File: sketch.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+function getSize() {
+    const input = prompt("Size of the board");
+
+    const message = document.querySelector("#text");
+
+    if(input === ""){
+        message.innerHTML = "Please provide a number";
+    }
+    else if (input < 0 || input > 100){
```
Comment (roopaljasnani) [inline sketch.js]: should this be input <= 0 ?

File: sketch.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+function colorDiv() {
+    if (color === "random"){
+        this.style.backgroundColor = `hsl(${Math.random() * 360} , 100% , 50%)`;
+    }
+    else if (color === "white"){
+        this.style.backgroundColor = "white";
+    }
+    else{
+        this.style.backgroundColor = "black";
+    }
```
Comment (roopaljasnani) [inline sketch.js]: you can simply write


```js
else {
  this.style.backgroundColor = color;
}
```

File: index.html (hunk 1 above)
Comment (Sushilkumar3255) [reply index.html]: okay mam. Done

File: sketch.js (hunk 2 above)
Comment (Sushilkumar3255) [reply sketch.js]: Yaa mam. Done

File: sketch.js (hunk 3 above)
Comment (Sushilkumar3255) [reply sketch.js]: okay mam. Done

File: sketch.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+function colorDiv() {
+    if (color === "random"){
+        this.style.backgroundColor = `hsl(${Math.random() * 360} , 100% , 50%)`;
+    }
+    else if (color === "white"){
+        this.style.backgroundColor = color;
+    }
+    else{
+        this.style.backgroundColor = "black";
+    }
```
Comment (roopaljasnani) [inline sketch.js]: You dont need both the `else if` here

Only - 
``` TSX
else {
  this.style.backgroundColor = color;
}
```

**Review by roopaljasnani — APPROVED** [review]
Good to merge after this last comment is resolved

File: sketch.js (hunk 4 above)
Comment (Sushilkumar3255) [reply sketch.js]: okay mam. Done

**Pull request merged**